hack/generator/pkg/astmodel/resource.go

```diff
@@ -140,9 +140,7 @@ func (definition *ResourceType) AsDeclarations(codeGenerationContext *CodeGenera
 		})
 	}
 
-	if description != nil {
-		addDocComment(&comments, *description, 200)
-	}
+	addDocComments(&comments, description, 200)
```

Comment:
These seem wide? (I'd say the 120 ones are pretty wide too - there's no need for comments to be as wide as the maximum allowed line width for code. But that might just be me!)

Reply:
These wide ones are to handle URLs - if the width is too narrow, the URL gets wrapped to the next line.